page_objects/geography/geographyLandingPage.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    url: "http://localhost:20000/geography/",
+    elements: {
+        areatypes: {
+            selector: ".area-type",
+            locateStrategy: 'css selector'
+        },
+        areatypes0: {
```

Comment:
Similar to above, I think selectors for 3 specific area types might be a bit fragile. It's possible better to loop through the area types there, rather than specific selectors for each area type we currently have.